api/v1/checks.go

```diff
@@ -958,6 +957,29 @@ type Smb struct {
 	SmbCheck `yaml:",inline" json:",inline"`
 }
 
+type EC2Check struct {
+	Description   `yaml:",inline" json:",inline"`
+	AccessKeyID   kommons.EnvVar            `yaml:"accessKeyID" json:"accessKeyID,omitempty"`
+	SecretKey     kommons.EnvVar            `yaml:"secretKey" json:"secretKey,omitempty"`
```

Comment:
Can there be any scenario when accessKey or SecretKey is not needed. If not I think we should make them as required fields. Also if they can be empty I think it would be better to use `*kommons.EnvVar` so before fetching the check we see if it is empty or not

Reply:
You have to log into aws to perform this check, so the credentials will always need to be provided.

Reply:
Yeah so in the when both are required fields I suggest we remove the `omitempty` that way user would have to provide them before the CR can be created